### PR TITLE
Finish patch votes

### PR DESCRIPTION
**Pull request by HartThomas**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -205,3 +205,55 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+      .patch("/api/reviews/2")
+      .send({ inc_votes: "notAVote" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid input type");
+      });
+  });
+  test("status - 404, if given an invalid id", () => {
+    return request(app)
+      .patch("/api/reviews/9001")
```
Comment (yalicee) [inline __tests__/app.test.js]: Should be 400

**Review by yalicee — CHANGES_REQUESTED** [review]
V good so far just missing a error handling for when a review is not found 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (HartThomas) [reply __tests__/app.test.js]: I thought 404 meant not found. According to the notes i think it should be 404:
'/api/resource/999999999 -> resource that does not exist: 404 Not Found'

**Review by yalicee — APPROVED** [review]
🔥 🔥 🔥 🔥 🔥

**Pull request merged**